### DISCUSSION TITLE
Grafana 8.4.3 Alert Message Customize

### DISCUSSION THREAD

**Discussion by Geek-Joey** [grafana/grafana#46350] [Alerting]
I want to customize the format of the message ,  how should i do it , thank you.

 I only set the message's  summary, but the  output  default message is :
```shell
Firing
Value: [no value]
Labels:
alertname=rule_hb_vehicle motorecognition 
datasource_uid=uhj5myY7z 
ref id=AAnnotations:
summary=hb_vehicle_motorecognition过去1小时数据量小于0
Source:http://localhost:3000/alerting/bQafWsY7z/edit 
Silence:http://localhost:3000/alerting/silence/new?
alertmanager=grafana&matcher=alertname%3Drule_hb_vehicle_motorecognition&matcher=datasource_uid%3Duhj5myY7z&matcher=ref_id%3DA
``` 
the result i want : 
```shell
Firing
Value: [no value]
Labels:
alertname=rule_hb_vehicle motorecognition 
datasource_uid=uhj5myY7z 
ref id=AAnnotations:
summary=hb_vehicle_motorecognition过去1小时数据量小于0
```
![image](https://user-images.githubusercontent.com/80373700/157402289-eed7b13f-1466-45aa-b166-2fb69f0a100e.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
